android/libraries/rib-router-navigator/src/test/kotlin/com/uber/rib/core/StackRouterNavigatorTest.kt

```diff
@@ -455,10 +438,7 @@ class StackRouterNavigatorTest {
     routerNavigator.pushState(TestState.STATE_1, attachTransition1, detachTransition1)
     routerNavigator.pushState(TestState.STATE_2, attachTransition2, detachTransition2)
     routerNavigator.pushState(
-      TestState.STATE_3,
-      RouterNavigator.Flag.NEW_TASK,
-      attachTransition3,
-      detachTransition3
+      TestState.STATE_3, RouterNavigator.Flag.NEW_TASK, attachTransition3, detachTransition3
```

Comment:
IMO, changes that just format code should be avoided. This makes the PR harder to review.

Reply:
Will remove similar changes